srcpkgs/websocat/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'websocat'
 pkgname=websocat
-version=1.6.0
-revision=2
+version=1.9.0
+revision=1
 build_style=cargo
 configure_args="--features=ssl"
-hostmakedepends="pkg-config"
+hostmakedepends="perl pkg-config"
```

Comment:
Do you remember what crate failed without `perl`? This looks slightly suspicious because some packages that built their own internal OpenSSL pull in perl, but this seems to be dynamically linked to our OpenSSL still.

Reply:
I was initially trying to update websocat to 1.8.1 before realizing that the yanked crates would be a problem. I can't remember which crate failed without `perl` in v1.8.1, but in v1.9.0 none of them need it. I've removed it in the latest push.